ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -17,6 +17,8 @@ images:
       as:
       - build
   to: console
+- from: src
```

Comment:
Perfer to add this mapping in the `promotion` stanza so you don't do another image build https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#promotionadditional_images

Reply:
That's super useful, thanks!